src/main/java/com/vesta/repository/SubjectRepository.java

```diff
@@ -12,9 +14,11 @@
 
     Optional<SubjectEntity> findById(Long id);
 
-    void deleteById(Long id);
-
     List<SubjectEntity> findAll();
 
     List<SubjectEntity> findAllByFloorId(Long floorId);
+
+    @Modifying
+    @Query("delete from SubjectEntity s where s.id in ?1")
```

Comment:
what mean 1 in "delete from SubjectEntity s where s.id in ?1"

Reply:
parce que cette valeur doit être, que sans elle cela ne fonctionne pas!!

Reply:
You can use: ':ids' instead of '?1'